### ISSUE TITLE
Error when installing latest version

### ISSUE THREAD

**Issue by rwaldron** [nib-edit/Nib#128]
```
$ yarn add nib-core@2.6.2 --dev
yarn add v1.22.4
[1/4] 🔍  Resolving packages...
error Couldn't find package "nib-ui-select@0.0.1" required by "nib-core@2.6.2" on the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by jpuri** [comment]
Hey @rwaldron : in newer versions of nib-core I have implemented code splitting to move heavier `nib-ui-select` to different chunk. Might be something got messed up in this version - I will check this soon.

Thanks a lot for reporting this.

**Comment by jpuri** [comment]
Hey @rwaldron : I just published `nib-ui-select@0.0.1` to npm and that fixes this issue.

**Comment by rwaldron** [comment]
Thanks!!
